deployment/templates/fluentd.yaml

```diff
@@ -58,20 +44,15 @@ objects:
             component: "fluentd"
         spec:
           serviceAccountName: aggregated-logging-fluentd
+          nodeSelector:
+            logging-infra: fluentd
           containers:
           - name: fluentd-elasticsearch
-            image: logging-fluentd
+            image: ${IMAGE_PREFIX}logging-fluentd
```

Comment:
if we leave off ${IMAGE_PREFIX} it will try to pull in the logging-fluentd image from `library/logging-fluentd`.  Unless I'm mistaken that seems to be different than our other containers trying to pull from `openshift/` if they aren't dev builds.